how-to-guides/cloud-services/azure/setting-up-dns-private-resolver.md

```diff
@@ -0,0 +1,69 @@
+# Azure DNS Private Resolver
+
+Setting up a DNS Private Resolver so that you can access the Azure Private DNS zones connected to your VNET.
```

Comment:
What is VNET? Preferably spell it out first, then use the acronym later

Reply:
okay, will fix that right away.